src/V3Param.cpp

```diff
@@ -542,12 +542,28 @@ class ParamProcessor final {
             hash.insert(V3Os::trueRandom(64));
         }
     }
+    void replaceRefs(AstNode* const nodep, const AstClass* const oldClassp,
+                     AstClass* const newClassp) {
+        if (!nodep) return;
+        if (AstClassRefDType* classRefp = VN_CAST(nodep, ClassRefDType)) {
+            if (classRefp->classp() == oldClassp) { classRefp->classp(newClassp); }
+        }
+        replaceRefs(nodep->op1p(), oldClassp, newClassp);
+        replaceRefs(nodep->op2p(), oldClassp, newClassp);
+        replaceRefs(nodep->op3p(), oldClassp, newClassp);
+        replaceRefs(nodep->op4p(), oldClassp, newClassp);
+        replaceRefs(nodep->nextp(), oldClassp, newClassp);
```

Comment:
```suggestion
        if (op1p()) replaceRefsRecurse(nodep->op1p(), oldClassp, newClassp);
        if (op2p()) replaceRefsRecurse(nodep->op2p(), oldClassp, newClassp);
        if (op3p()) replaceRefsRecurse(nodep->op3p(), oldClassp, newClassp);
        if (op4p()) replaceRefsRecurse(nodep->op4p(), oldClassp, newClassp);
        if (nextp()) replaceRefsRecurse(nodep->nextp(), oldClassp, newClassp);
```

It's faster to do the if before recursing.